### ISSUE TITLE
[IMPROVEMENTS]: 

### ISSUE THREAD

**Issue by shkatz** [typical-developers/oaklands-feedback#132]
### Roblox Username

shkatz

### What do you think can have quality-of-life improvements?

filter conveyor not filtering ores properly

### How do you think things can be improved?

if a rock node is assigned to a filter conveyor, then any ore containing that type of rock will pass through it.
but, if a ore is assigned to a filter conveyor, only that ore containing its rock note type can pass through it.
that means if i want to filter only copper ore from all the stuff that is mined i would need every variant of copper ore assigned to 18 different filter conveyor.
and if i want to filter every ore from every stone then i would need 450 filter conveyors or $126,000 just to filter ores from stones

to summarize, if i assign dark slate to a filter conveyor, why cant it only be dark slate itself that passes through it. why everything dark slate related.

change it to do that or make it a second filter mode pleas. this has been irritating me for 3 days.

https://github.com/typical-developers/oaklands-feedback/assets/150176103/37aede1a-5702-449d-82a6-48e35c8e170a
